src/domain/entities/Settings.h

```diff
@@ -21,6 +21,8 @@ extern "C"
         {
         private:
             String _deviceName;
+            String _wifiSSID;
+            String _wifiPassword;
```

Comment:
Please rename `wifiSSID` to `webuiWifiSSID` and `wifiPassword` to `webuiWifiPassword`.